test/remote/gateways/remote_shift4_test.rb

```diff
@@ -78,6 +78,7 @@ def test_successful_purchase_with_stored_credential
       initial_transaction: true,
       reason_type: 'recurring'
     }
+    @options[:merchant_time_zone] = 'EST'
```

Comment:
Can we add an expect here to verify the timezone is set properly?

Reply:
Just added a unit test for this. Thanks for the suggestion!